static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeDetector.kt

```diff
@@ -63,11 +63,13 @@ class AutoDisposeDetector: Detector(), SourceCodeScanner {
 
     private val reactiveTypes = mutableSetOf(OBSERVABLE, FLOWABLE, PARALLEL_FLOWABLE, SINGLE, MAYBE,
         COMPLETABLE)
+
+    const val PROPERTY_FILE = "gradle.properties"
```

Comment:
nit: private

Reply:
Will change to `internal`. Reusing this in tests.

Reply:
06d7324829c34b6fcbb0ceb34b62597f45d2a7e8